.buildkite/scripts/pipelines/chromium_linux_build/issue_feedback/entry.js

```diff
@@ -0,0 +1,373 @@
+
+      const binaryChecksum = await getSha256Hash(`headless_shell-linux_${arch}/headless_shell`);
+
+      config[`linux_${arch}`] = {
+        archiveFilename: match,
+        archiveChecksum,
+        binaryChecksum,
+      };
+    })
+  );
+
+  console.log('--Modifying paths.ts file\n');
+
+  await $(
+    'npx',
+    [
+      '--',
+      'jscodeshift',
+      '--extensions=ts',
+      '--parser=tsx',
+      '--fail-on-error',
+      `--transform`,
+      resolve(
+        buildRoot,
+        '.buildkite/scripts/pipelines/chromium_linux_build/issue_feedback/transform_path_file.js'
+      ),
+      '../paths.ts',
+      `--updateConfig=${JSON.stringify(config)}`,
+      `--chromiumVersion=${chromiumVersion}`,
+    ],
+    { printToScreen: true }
+  );
+
+  await $('git', ['add', '../paths.ts']);
+  await $('git', ['commit', '-m', 'chore: update chromium paths']);
+
+  // create clean branch based off of main
+  await $('git', ['checkout', 'main']);
+  await $('git', ['checkout', '-b', branchName]);
+  // cherry-pick the two commits we made to the temp branch
+  await $('git', ['cherry-pick', `${branchName}_temp~1`, `${branchName}_temp~0`], {
+    printToScreen: true,
+  });
+
+  await $('git', ['push', 'origin', branchName], {
+    printToScreen: true,
+  });
+
+  openPRForVersionUpgrade = await $(
+    'gh',
+    [
+      'pr',
+      'create',
+      '--title',
+      prTitle,
+      '--body',
+      `Closes #${process.env.GITHUB_ISSUE_NUMBER} \n\n This PR updates puppeteer to version \`${process.env.PUPPETEER_VERSION}\` and updates the chromium paths to the latest known good version for windows and mac where the chromium revision is \`${chromiumRevision}\` and version is \`${chromiumVersion}\`.\nFor linux a custom build was triggered to build chromium binaries for both x64 and arm64. \n\n\n **NB** This PR should be tested before merging it in as puppeteer might have breaking changes we are not aware of`,
+      '--base',
+      'main',
+      '--head',
+      branchName,
+      '--label',
+      'release_note:skip',
+      '--label',
+      'Team:SharedUX',
+      '--label',
+      'backport',
+      '--label',
+      'Feature:Reporting:Framework',
+      '--assignee',
+      process.env.GITHUB_ISSUE_TRIGGER_USER,
+    ],
+    {
+      printToScreen: true,
+    }
+  );
+
+  console.log('---Providing feedback to issue \n');
+
+  await $(
+    'ts-node',
+    [
+      resolve(buildRoot, '.buildkite/scripts/lifecycle/comment_on_pr.ts'),
+      '--message',
+      String.raw`@${process.env.GITHUB_ISSUE_TRIGGER_USER} a PR has been created to upgrade puppeteer to the requested version at ${openPRForVersionUpgrade}`,
+      '--issue-number',
+      process.env.GITHUB_ISSUE_NUMBER,
+      '--repository',
+      process.env.GITHUB_ISSUE_BASE_REPO,
+      '--repository-owner',
+      process.env.GITHUB_ISSUE_BASE_OWNER,
+      '--context',
+      'chromium-linux-build-diff',
+      '--clear-previous',
+    ],
+    { printToScreen: true }
+  );
```

Comment:
This step will fail to run, whilst testing it in this PR because we made some changes to the `comment_on_pr.ts` file that would not be available on main, till the branch is merged in, however, we can verify that it works because the same script is invoked in the beginning of this file and is used to leave a comment for a pre-exisiting PR, which we have tested and can be seen the referenced test issue.

Reply:
Since we extracted the changes to `comment_on_pr.ts` out this comment is not relevant anymore.